src/AtmosphereModels/microphysics_interface.jl

```diff
@@ -65,7 +65,7 @@ Return the microphysical velocities associated with `microphysics` and `name`.
 
 Must be either `nothing`, or a NamedTuple with three components `u, v, w`.
 """
-@inline microphysical_velocities(microphysics::Nothing, name) = nothing
+@inline microphysical_velocities(microphysics::Nothing, name, microphysical_fields) = nothing
```

Comment:
I believe the main problem which caused all tests to fail is that you changed the signature of this function, without changing its uses, for example https://github.com/NumericalEarth/Breeze.jl/blob/07e2ec9e953b82540382cd6e0b05c1358b416b96/src/AtmosphereModels/dynamics_kernel_functions.jl#L139 but there are more.  Also, I don't understand why this was changed in the first place, in the rest of this PR it doesn't look like you make use of this 3-argument signature anywhere.